release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -490,6 +490,70 @@ For more information, see xref:../registry/configuring_registry_storage/configur
+
+[id="ocp-4-17-storage-azure-file-snapshot"]
+==== Azure File CSI supports snapshots (Technology Preview)
+{product-title} 4.17 introduces volume snapshot support for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator. This capability is supported as a Technology Preview feature.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi.adoc#csi-drivers-supported_persistent-storage-csi[CSI drivers supported by {product-title}] and xref:../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc[CSI volume snapshots].
+
+[id="ocp-4-17-storage-multi-vcenter-support"]
+==== Multiple vCenter support for vSphere CSI (Technology Preview)
+{product-title} v4.17 introduces the ability to deploy {product-title} across multiple vSphere clusters (vCenters). This feature is supported with Technology Preview status.
+
+Multiple vCenters can only be configured during installation. The maximum number of supported vCenter clusters is three.
+
+// For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-multi-vcenter-support-overview_persistent-storage-csi-vsphere[Multiple vCenter support for vSphere CSI] and xref:../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
+
+[id="ocp-4-17-storage-selinuxmount-dev-preview"]
+==== RWX/RWO SELinux Mount (Developer Preview)
+Pods might take a very long time to start when the volume contains a large number of files. To avoid SELinux labeling issues while keeping SELinux confining, you can enable the ReadWriteMany/ReadWriteOnce (RWX/RWO) SELinux Mount feature. Be advised that the RWX/RWO SELinux Mount feature is a Developer Preview feature. It is not supported by Red Hat, and you should not enable this feature set on production or clusters that you plan to maintain over time.
+
+:FeatureName: RWX/RWO SELinux Mount
+include::snippets/technology-preview.adoc[]
+
+For more information about the RWX/RWO SELinux Mount feature, including how to enable it, see link:https://access.redhat.com/articles/7087028[RWX/RWO SELinux Mount feature Knowledge Centered Service article].
+
+[id="ocp-4-17-storage-migrating-cns-vols-between-datastores-dev-preview"]
+==== Migrating CNS volumes between datastores with CLI (Developer Preview)
+In {product-title} 4.17, if you are running out of space in your current datastore, or want to move to a more performant datastore, you can migrate volumes between datastores. Be advised that this feature is a Developer Preview feature. It is not supported by Red Hat.
+
+:FeatureName: Migrating CNS Volumes Between Datastores
+include::snippets/technology-preview.adoc[]
+
+//Need some kind of link to KCS or other form of support???
```

Comment:
@sferich888 what do we say here? Do we even need this section if we can't link to anything meaningful?  Can we link to Github PR that implemented this -  https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/242 ?

Reply:
I would like to the code or a readme in the repo that explains who to use the tool. But because this is developer preview; how fare Engineering takes documenting this is really up to them; what we have here is all we need for this to be denoted as developer preview.

Reply:
@gnufied @sferich888 We have KCS article linked now: https://access.redhat.com/articles/7086450

Reply:
That KCS is supposed to be private. I am going to be writing a README on github, which we can use here.

Reply:
@gnufied OK. If we prefer something else, no problem.

I just used this KCS because @gcharot provided it https://redhat-internal.slack.com/archives/GK0DA0JR5/p1726137660979849.

Reply:
Waiting for https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/252 to be merged, so as we can refer it here.

Reply:
Yeh the KCS is private and use internal language so let's use @gnufied readme instead

Reply:
Link is ready -  https://github.com/openshift/vmware-vsphere-csi-driver-operator/blob/master/docs/cns-migration.md

Reply:
Done.